pkg/vault/contracts/CompositeLiquidityRouter.sol

```diff
@@ -642,9 +625,10 @@ contract CompositeLiquidityRouter is ICompositeLiquidityRouter, BatchRouterCommo
             })
         );
 
-        // Remove the underlying amount from `_currentSwapTokenInAmounts` and add the wrapped amount.
+        // Remove the underlying token from the tokensIn and set the amount as zero because we took it here.
+        // We will take other tokens at the end of the calculation.
```

Comment:
```suggestion
        // Remove the underlying token from `_currentSwapTokensIn` and zero out the amount, as these tokens were paid
        // in advance and wrapped. Remaining tokens will be transferred in at the end of the calculation.
```
Is this correct? I wasn't sure what "taking the token" meant in the original.

Reply:
Was my suggestion inaccurate? I still think "took" is ambiguous.

Reply:
Sorry, I misunderstood at first